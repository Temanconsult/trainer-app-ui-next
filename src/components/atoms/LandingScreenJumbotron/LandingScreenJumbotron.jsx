import React from 'react'
import { Jumbotron, Button } from 'react-bootstrap'

const LandingScreenJumbotron = () => {
  return (
    <Jumbotron>
      <h1>Learn with Ease and at your own pace</h1>
      <p>
        This is a simple hero unit, a simple jumbotron-style component for
        calling extra attention to featured content or information.
      </p>
      <p>
        <Button variant="primary">Learn more</Button>
      </p>
    </Jumbotron>
  )
}

export default LandingScreenJumbotron
