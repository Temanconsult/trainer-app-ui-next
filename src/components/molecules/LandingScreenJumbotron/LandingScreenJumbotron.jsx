import React from 'react'
import { Media, Button, Col, Row, ResponsiveEmbed } from 'react-bootstrap'
import tutors from '../../../assets/images/tutor.svg'

const LandingScreenJumbotron = () => {
  return (
    <Media className="landing-screen-jumbotron mt-4">
      <Media.Body>
        <Row>
          <Col sm={12} md={12} lg={5} className="call-out">
            <h1>Learn with Ease and at your own pace</h1>
            <p className="mt-4">
              Connect with the best minds in any field of study or professional
              cources of your choice and learn the best way you can.
            </p>
            <p>
              <Button variant="primary">Learn more</Button>
            </p>
          </Col>
          <Col sm={0} md={0} lg={7} className="d-none d-md-block">
            <ResponsiveEmbed aspectRatio="16by9">
              <embed type="image/svg+xml" src={tutors} />
            </ResponsiveEmbed>
          </Col>
        </Row>
      </Media.Body>
    </Media>
  )
}

export default LandingScreenJumbotron
