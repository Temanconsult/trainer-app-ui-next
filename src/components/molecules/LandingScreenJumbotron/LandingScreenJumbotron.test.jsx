import React from 'react'
import { render } from '@testing-library/react'
import LandingScreenJumbotron from '.'

describe('LandingScreenJumbotron', () => {
  it('creates landing screen jumbotron', () => {
    const { asFragment } = render(<LandingScreenJumbotron />)
    expect(asFragment()).toMatchSnapshot()
  })
})
