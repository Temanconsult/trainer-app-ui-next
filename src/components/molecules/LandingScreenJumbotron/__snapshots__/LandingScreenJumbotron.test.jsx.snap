// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingScreenJumbotron creates landing screen jumbotron 1`] = `
<DocumentFragment>
  <div
    class="landing-screen-jumbotron mt-4 media"
  >
    <div
      class="media-body"
    >
      <div
        class="row"
      >
        <div
          class="call-out col-lg-5 col-md-12 col-sm-12"
        >
          <h1>
            Learn with Ease and at your own pace
          </h1>
          <p
            class="mt-4"
          >
            Connect with the best minds in any field of study or professional cources of your choice and learn the best way you can.
          </p>
          <p>
            <button
              class="btn btn-primary"
              type="button"
            >
              Learn more
            </button>
          </p>
        </div>
        <div
          class="d-none d-md-block col-lg-7 col-md-0 col-sm-0"
        >
          <div
            class="embed-responsive embed-responsive-16by9"
          >
            <embed
              class="embed-responsive-item"
              src="file mock"
              type="image/svg+xml"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
